projects/management-software-sync/backend/samples/schema/storages.json

```diff
@@ -1,6 +1,6 @@
 [
-    {
-        "id_deposito": "",
-        "descr_dep": ""
-      }
-]
\ No newline at end of file
+  {
+    "id_deposito": "",
+    "descr_dep": ""
```

Comment:
I assume that is `descr_depositvo`. In the relocation, i'll prefer call it just `description`.

Reply:
Agree